src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,131 @@
+$productMarginY: 23px;
+$productMarginX: 24px;
+$productWidth: 200px;
+
+@function calcWidth ($cardWidth, $cardMarginX) {
+  @return ($cardMarginX * 8 + $cardWidth * 4);
+}
+
+@mixin fontSizeWeight($size, $weight: 400) {
+  font-size: $size;
+  font-weight: $weight;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.products {
+  display: flex;
+  width: calcWidth($productWidth, $productMarginX);
+  flex-wrap: wrap;
+
+}
+
+.page__products {
+  margin: (50px - $productMarginY) auto 72px;
+}
+
+.products__card {
+  margin: $productMarginY $productMarginX;
+}
+
+.product-card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: $productWidth;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: white;
+
+  &__image {
+    margin: 0 3px 36px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__name {
+    @include fontSizeWeight(12px, 500);
+    line-height: 150%;
+
+    color: $color-black;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-black;
+
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+    flex-grow: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    display: flex;
+    align-items: flex-end;
+
+    color: #616070;
+  }
+
+  &__price-dollar {
+    @include fontSizeWeight(16px, 700);
+    line-height: 18px;
+    text-align: right;
+    color: $color-black;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    text-align: center;
+    width: 100%;
+
+    background-color: $color-light-blue;
+    border-radius: 5px;
+
+    text-decoration: none;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)